### ISSUE TITLE
onMouseUp stops all propagation when selecting text

### ISSUE THREAD

**Issue by Johns-Chris** [bfwg/ngx-drag-scroll#129]
## Expected Behavior
When selecting text, either by `CTRL+A`, or double-click a line of text to select, user should be able to deselect the text easily; by clicking anywhere on the window once.

## Current Behavior
When selecting text, either by `CTRL+A`, or double-click a line of text to select, user has a difficult time deselecting the text.

## Possible Solution
I was able to prevent the issue by commenting out lines 479, and 488, but I know that this may not be an ideal solution...
```
...
  onMouseUp(e: MouseEvent) {
    // e.preventDefault();
    if (this.isPressed) {
      this.isPressed = false;
      if (!this.snapDisabled) {
        this.locateCurrentIndex(true);
      } else {
        this.locateCurrentIndex();
      }
    }
    // return false;
  }
...
```

## Steps to Reproduce

1. Visit the [demo page](https://bfwg.github.io/ngx-drag-scroll/)
2. Press `CTRL+A` select-all, or double-click a line of text to select
3. Try to deselect text

**Comment by bfwg** [comment]
Hi @Johns-Chris , thanks for reporting this. I've tested your solution and it doesn't break anything for me.

**Comment by bfwg** [comment]
The fix is in 1.7.7. Please let me know if you find any regression. 

Thanks!
